From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Clark <tjclark@wpi.edu>
Date: Fri, 25 Jul 2014 09:59:20 -0400
Subject: [PATCH 0202/6262] Fixed CANJaguar percent scaling issue [artf2637]

Percent mode used to be scaled incorrectly in Java because the
max output voltage was initialized to a wrong value.

Change-Id: If88bb2f1d198e13b1724afc62a522fbf91a14b5b
---
 .../src/main/java/edu/wpi/first/wpilibj/CANJaguar.java          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java b/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
index aac31d8b38658f320b50b5cf8f65f194d07330eb..75f4fa72e8ef09db7935a276770bf7bfdb5f2f4e 100644
--- a/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
+++ b/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
@@ -1695,7 +1695,7 @@ public class CANJaguar implements MotorSafety, PIDOutput, SpeedController, LiveW
 	LimitMode m_limitMode = LimitMode.SwitchInputsOnly;
 	double m_forwardLimit = 0.0;
 	double m_reverseLimit = 0.0;
-	double m_maxOutputVoltage = 30.0;
+	double m_maxOutputVoltage = kApproxBusVoltage;
 	double m_voltageRampRate = 0.0;
 	float m_faultTime = 0.0f;
 
